google/cloud/bigtable/table.py

```diff
@@ -782,6 +783,177 @@ def mutations_batcher(self, flush_count=FLUSH_COUNT, max_row_bytes=MAX_ROW_BYTES
+                -  ``(name:howl) AND (start_time < \"2020-05-28T14:50:00Z\")``
+                		--> The Backup name contains the string "howl" and
+                		the Backup start time is before 2020-05-28T14:50:00Z.
+                -  ``size_bytes > 10000000000`` --> The Backup size is greater
+                		than 10GB
+
+        :type order_by: str
+        :param order_by: (Optional) An expression for specifying the sort order
+        				 of the results of the request. The string value should
+        				 specify one or more fields in ``Backup``. The full
+        				 syntax is described at https://aip.dev/132#ordering.
+
+                		 Fields supported are: \* name \* source\_table \*
+                		 expire\_time \* start\_time \* end\_time \*
+                		 size\_bytes \* state
+
+                		 For example, "start\_time". The default sorting order
+                		 is ascending. To specify descending order for the
+                		 field, a suffix " desc" should be appended to the
+                		 field name. For example, "start\_time desc". Redundant
+                		 space characters in the syntax are insigificant. If
+                		 order_by is empty, results will be sorted by
+                		 ``start_time`` in descending order starting from
+                		 the most recently created backup.
+
+        :type page_size: int
+        :param page_size: (Optional) The maximum number of resources contained
+        				  in the underlying API response. If page streaming is
+        				  performed per-resource, this parameter does not
+        				  affect the return value. If page streaming is
+        				  performed per-page, this determines the maximum
+        				  number of resources in a page.
+
+        :rtype: :class:`~google.api_core.page_iterator.Iterator`
+        :returns: Iterator of :class:`~google.cloud.bigtable.backup.Backup`
+                  resources within the current Instance.
+        :raises: :class:`ValueError <exceptions.ValueError>` if one of the
+                 returned Backups' name is not of the expected format.
+        """
+        cluster_id = cluster_id or "-"
+
+        backup_list_pb = self._instance._client.table_admin_client.list_backups(
+            self._instance.name + "/clusters/" + cluster_id,
+            filter_=filter_,
+            order_by=order_by,
+            page_size=page_size,
+        )
+
+        result = []
+        for backup_pb in backup_list_pb:
+            backup_id = backup_pb.name.split("/")[-1]
+            backup_cluster_id = backup_pb.name.split("/")[-3]
+            backup_expire_time = backup_pb.expire_time
+            result.append(self.backup(
+                backup_id,
+                cluster_id=backup_cluster_id,
+                expire_time=backup_expire_time
+            ))
+
+        return result
+
+    def restore(
+        self, table_id, cluster_id=None, backup_id=None, backup_name=None
```

Comment:
Another suggestion, instead of `cluster_id`, `backup_id`, `backup_name` use param `source` which could be either full backup_name (`projects/<project>/instances/<instance>/clusters/<cluster>/backups/<backup>`) or `Backup` instance
```py
from google.cloud.bigtable.backup import _BACKUP_NAME_RE
backup_name = ""
if not source:
  raise ValueError("")
if isinstance(source, Backup):
  backup_name = Backup.name
elif isinstance(source, str):
  match_backup_name = _BACKUP_NAME_RE.match(source)
  if match_backup_name is None:
    raise ValueError("")
  backup_name = source
else: raise ValueError("source should be ...")
```